src/components/graph/nodes/network-modification-node.js

```diff
@@ -165,7 +204,7 @@ const NetworkModificationNode = (props) => {
                         : classes.networkModification
                 }
             >
-                <div className={getClassForBanner(props.data.buildStatus)}>
+                <div className={getClassForBanner(props.data.buildStatusLocal)}>
                     {props.data.buildStatus === 'BUILDING' && (
```

Comment:
buildStatusComputed, the progress bar will never display

Reply:
Indeed, I just fixed this